paddle/fluid/inference/api/paddle_inference_api.h

```diff
@@ -259,11 +259,20 @@ struct AnalysisConfig : public NativeConfig {
     kExclude   // Specify the disabled passes in `ir_passes`.
   };
 
+  void SetIncludeMode() {
+    ir_mode = IrPassMode::kInclude;
+    ir_passes = {"infer_clean_graph_pass"};
+    ir_mkldnn_passes = {"infer_clean_graph_pass"};
+  }
+
   // Determine whether to perform graph optimization.
   bool enable_ir_optim = true;
   // Manually determine the IR passes to run.
   IrPassMode ir_mode{IrPassMode::kExclude};
+  // passes to be excluded/included
   std::vector<std::string> ir_passes{"embedding_fc_lstm_fuse_pass"};
+  // passes to be excluded/included when MKL-DNN is enabled
+  std::vector<std::string> ir_mkldnn_passes{"infer_clean_graph_pass"};
```

Comment:
Why should line275 `ir_mkldnn_passes{"infer_clean_graph_pass"}`?

Reply:
No. Done.